terraform/implementation/ecs/unlock.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Did you mean to save this file?  If so, perhaps add a note for the partner to inform them this is the command to unlock the terraform state file.  Otherwise, we may want to consider removing.

Reply:
I don't mind leaving it as a helper script, mostly for me and not clients :smile:, but I also don't care if we remove it.

Reply:
I removed the script